### ISSUE TITLE
White border around admin button

### ISSUE THREAD

**Issue by Pips801** [UploadX/UploadX#3]
https://pips.rocks/L9Z

**Comment by NullOrNaN** [comment]
Yeah, I did that to differentiate the admin section from the user section - it's still an "alpha" design for this.

Suggestions on what to do to make the admin  section stand out easily would be great.

**Comment by Pips801** [comment]
It doesn't really need to stand out though. It's a button that only the admin sees, only if he's logged in.

**Comment by NullOrNaN** [comment]
Hmm.. I see your point, however, an admin might want to know what are admin functions and what's not (hence doing that).

I'm open to reverting it though
